### ISSUE TITLE
Fix links in READMEs

### ISSUE THREAD

**Issue by micimize** [zino-app/graphql-flutter#244]
The READMEs still have a bunch of placeholders from the original build and coverage work
```md
[build-status-badge]: https://api.cirrus-ci.com/github/truongsinh/graphql-flutter.svg
[build-status-link]: https://cirrus-ci.com/github/truongsinh/dart-uuid/master
[coverage-badge]: https://codecov.io/gh/truongsinh/graphql-flutter/branch/master/graph/badge.svg
[coverage-link]: https://codecov.io/gh/truongsinh/graphql-flutter
```

**Comment by micimize** [comment]
closed by 42a70b1

**Comment by micimize** [comment]
though one issue is links in master vs beta
